crates/turborepo-lib/src/run/scope/change_detector.rs

```diff
@@ -109,8 +109,47 @@ impl<'a> GitChangeDetector for ScopeChangeDetector<'a> {
                         (
                             name.to_owned(),
                             PackageInclusionReason::All(AllPackageChangeReason::GitRefNotFound {
-                                from_ref: from_ref.clone(),
-                                to_ref: to_ref.clone(),
+                                from_ref,
+                                to_ref,
+                            }),
+                        )
+                    })
+                    .collect());
+            }
+            Err(ScmError::Path(err, _)) => {
```

Comment:
Can you deduplicate the generation of "all packages changed due to error" logic?